apps/heft/src/cli/actions/TestAction.ts

```diff
@@ -1,40 +1,115 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
```

Comment:
1. In the `heft-test-01` project folder, delete **ExampleTest.test.ts** and **ExampleTest.test.js.snap**
2. Run `heft test --clean`

**Expected:** Heft lets me work on my project.  I will choose to add tests whenever I'm ready.  I should NOT be expected to manually add custom settings to make Heft accept this absolutely normal starting point for a new project.  (It would be fine for Heft to print an informational message saying it didn't find any tests, though.)

**Actual:** The build fails with this message:
```
 ---- Test started ----
No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In C:\Git\rushstack\build-tests\heft-test-01
  33 files checked.
  testMatch: C:/Git/rushstack/build-tests/heft-test-01/lib/**/*.test.js - 0 matches
  testPathIgnorePatterns: \\node_modules\\ - 33 matches
  testRegex:  - 0 matches
Pattern:  - 0 matches
```
The log message does not clearly convey that it is an "error" (and there is no red colored output).  But the exit code fails the build.